frontend/app/components/redesign/components/InputField.tsx

```diff
@@ -83,6 +99,7 @@ export const InputField = forwardRef<HTMLInputElement, InputFieldProps>(
             <span
               id="input-error"
               role="alert"
+              aria-live="assertive"
```

Comment:
Consider `polite`. Assertive can be very noisy depending on when validation happens.